src/physics/reynolds_averaged_navier_stokes.h

```diff
@@ -0,0 +1,252 @@
+#ifndef __REYNOLDS_AVERAGED_NAVIER_STOKES__
+#define __REYNOLDS_AVERAGED_NAVIER_STOKES__
+
+#include "model.h"
+#include "navier_stokes.h"
+#include "euler.h"
+
+namespace PHiLiP {
+namespace Physics {
+
+/// Reynolds-Averaged Navier-Stokes equations. Derived from Navier-Stokes for modifying the stress tensor and heat flux, which is derived from PhysicsBase. 
+template <int dim, int nstate, typename real>
+class ReynoldsAveragedNavierStokesBase : public ModelBase <dim, nstate, real>
+{
+public:
+    using thermal_boundary_condition_enum = Parameters::NavierStokesParam::ThermalBoundaryCondition;
+    /// Constructor
+	ReynoldsAveragedNavierStokesBase
+(
+	    const double                                              ref_length,
```

Comment:
fix indentation

Reply:
fixed

Reply:
doesn't appear to be fixed, untab lines 18 and 19

Reply:
fixed